account_invoice_force_number/security/security.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
As per the [template](https://github.com/OCA/maintainer-tools/blob/master/template/module/security/some_model_security.xml#L1) you miss the encoding here